tests/test_reqif.py

```diff
@@ -531,6 +531,17 @@ def test_create_enum_value_attribute_on_requirements(
         assert attr.definition == definition
         assert isinstance(attr, reqif.EnumerationValueAttribute)
 
+    def test_create_enum_value_attribute_with_passing_values(
+        self, model: capellambse.MelodyModel
+    ):
+        req = model.oa.all_requirements[0]
+        dtdef = model.by_uuid("637caf95-3229-4607-99a0-7d7b990bc97f")
+
+        attr = req.attributes.create("Enum", values=dtdef.values[:1])
+
+        assert attr in req.attributes
+        assert attr.values == dtdef.values[:1]
```

Comment:
This looks a bit hacky, can we not just use simple `model.by_uuid` access instead of this `.values[:1]` thing here?

It also leads to a useless test if `dtdef.values` happens to become empty for some reason (like a bug that leads to the values not being found and an empty list returned instead).